src/anarcii/inference/model_loader.py

```diff
@@ -39,7 +39,7 @@ def _load_params(self):
         else:
             raise ValueError(
                 "Invalid mode specified. Choose either 'speed' or 'accuracy' or "
-                "'shark'."
+                "'shark/vnar/vhh'."
```

Comment:
```suggestion
                "'shark'/'vnar'/'vhh'."
```

Reply:
Alternatively, to make it really clear to the reader that `shark`, `vnar` and `vhh` are aliases, perhaps
```suggestion
                "'shark' (aliases 'vnar', 'vhh')."
```